### PR TITLE
improve readme

### PR DESCRIPTION
**Pull request by ngdream**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,39 @@
-# README - Fast Light Tool Kit (FLTK) Version 1.4.0
+![](https://github.com/ngdream/fltk/blob/bc61c1258ab8f2b8a9e36d9188c1f97d06e16a8f/share/fltkicon2.png)
```
Comment (Albrecht-S) [inline README.md:1]: This (linking to another repo) is not acceptable.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
+# FLTK - Fast Light Tool Kit (FLTK) 
+ The Fast Light Tool Kit ("FLTK", pronounced "fulltick") is across-platform C++ GUI toolkit for UNIX®/Linux® (X11 orWayland), Microsoft® Windows®, and macOS®.FLTK provides modern GUI functionality without the bloat andsupports 3D graphics via OpenGL® and its built-in GLUT emulation.
```
Comment (Albrecht-S) [inline README.md:3]: > is across-platform ...
> and macOS®.FLTK
> andsupports

(missing spaces)

**Comment by Albrecht-S** [comment]
Thank you for the PR, this will be considered, but...

- What is this "mc.txt" good for?
- There are a lot of typos in the text, could you please do a second check? See also some comments/review.
- We won't accept linking to a file in another repo
- Images must be approved by the FLTK Team.

That's not a final statement, it's just some things I noticed at a first glance. Looking forward to fixes... TIA.

**Comment by ngdream** [comment]
> Thank you for the PR, this will be considered, but...
> 
> * What is this "mc.txt" good for?
> * There are a lot of typos in the text, could you please do a second check? See also some comments/review.
> * We won't accept linking to a file in another repo
> * Images must be approved by the FLTK Team.
> 
> That's not a final statement, it's just some things I noticed at a first glance. Looking forward to fixes... TIA.

**Pull request closed without merging**